wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # if template is incomplete locally, delete and ask for a download
+    if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
> Over 90% of the users currently use the minimal template and not the other templates.

I don't think that's true. `wifiphisher` as an Evil Twin attack tool is more effective against networks with open authentication that may use captive portals. It would be great if we could provide one template for the most common scenarios (one for hotels, one for airports etc).

> It also saves a little bit of space.

The ideal would be to have 5-10 templates for different phishing scenarios. The rest of them can lie online. We should be fine in terms of storage. 

>  A big disadvantage of going with you method will be that check_template will no longer be a generic function and I feel it would be a dirty code.

That's true but i don't see why it's a disadvantage. Also, "dirty code" refers to code that's hard to maintain and that's not the case here. :)